applications/Preprocessor/output_impl.h

```diff
@@ -102,11 +105,20 @@ void outputMesh(Mesh<dimension>& mesh,
     }
     std::ofstream output(outputFileName.getValue());
     output << std::hexfloat;
-    output << "mesh 1" << std::endl;
+    size_t version = 2;
+    output << "mesh " << version << std::endl;
     output << mesh.getNumberOfNodes() << " " << mesh.getNumberOfElements()
            << " " << dimension;
     printOtherEntityCounts(output, mesh, Detail::tag<dimension - 1>{});
     output << std::endl;
+
+    output << "zones\n";
+    output << mesh.getZoneNames().size() << "\n";
+    for (size_t i = 0; i < mesh.getZoneNames().size(); i++) {
+        output << i << " " << mesh.getZoneNames()[i] << "\n";
```

Comment:
I'm wondering why you output i here? I don't see it in the new spec.

Reply:
ahh I thought the `#` in the spec was the number. Oops